index.html

```diff
@@ -30,18 +28,27 @@
             <a class=menu_item>Contact</a>
             <img class="menu_mail" src="./img/mail.png" alt="Email Icon">
         </nav>
-        <nav class="nav-toggle" id="navToggle">
-            <img src="./img/menu.png" alt="Hamburger Menu">
+        <nav id="navToggle" class="nav-toggle">
```

Comment:
- Your hamburger button isn't displayed, kindly fix that:
![Screen Shot 2021-09-07 at 6 03 31](https://user-images.githubusercontent.com/22258274/132334354-4a736634-25c4-4a79-80f3-36038e449d33.png)

Reply:
can you please check again because its fixed now

Reply:
@carlos-ssh